src/App.css

```diff
@@ -0,0 +1,96 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+.strike {
+  text-decoration: line-through;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+.area {
+  background-color: white;
+  width: 50.8%;
+  align-items: center;
+  text-align: center;
+  padding-bottom: 20px;
+}
+
```

Comment:
create css. files with each component and import styling from there and follow conventions like inputStyle for classesNames or todoHeadingStyle etc